debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java

```diff
@@ -2542,6 +2544,87 @@ public void shouldNotNullPositionBeforeOrAfterDefaultValue() {
         assertThat(table.columnWithName("ts_col5").defaultValue()).isEqualTo(isoEpoch);
     }
 
+    @Test
+    @FixFor("DBZ-2726")
+    public void shouldParseTimestampDefaultValue() {
+        // All the following default values for TIMESTAMP can be successfully applied to MySQL
+        String ddl = "CREATE TABLE my_table (" +
+                "ts_col01 TIMESTAMP DEFAULT '2020-01-02'," +
+                "ts_col02 TIMESTAMP DEFAULT '2020-01-02 '," +
+                "ts_col03 TIMESTAMP DEFAULT '2020-01-02:'," +
+                "ts_col04 TIMESTAMP DEFAULT '2020-01-02--'," +
+                "ts_col05 TIMESTAMP DEFAULT '2020-01-02 03'," +
+                "ts_col06 TIMESTAMP DEFAULT '2020-01-02 003'," +
+                "ts_col07 TIMESTAMP DEFAULT '2020-01-02 03:'," +
+                "ts_col08 TIMESTAMP DEFAULT '2020-01-02 03:04'," +
+                "ts_col09 TIMESTAMP DEFAULT '2020-01-02 03:004'," +
+                "ts_col10 TIMESTAMP DEFAULT '2020-01-02 03:04:05'," +
+                "ts_col11 TIMESTAMP DEFAULT '2020-01-02 03:04:05.6'," +
```

Comment:
Is also acceptable case when there is the period but no following numbers?

Reply:
It is an acceptable case both in MySQL and in Debezium when no numbers follow the period. I just added a test case for it.